code/game/objects/items/toys.dm

```diff
@@ -1049,6 +1050,31 @@
 	if(user.dropItemToGround(src))
 		throw_at(target, throw_range, throw_speed)
 
+/*
+ * Clockwork Watch
+ */
+
+/obj/item/toy/clockwork_watch
+	name = "steampunk watch"
+	desc = "A stylish steampunk watch made out of thousands of tiny cogwheels."
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "dread_ipad"
+	slot_flags = SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	var/cooldown = 0
+
+/obj/item/toy/clockwork_watch/attack_self(mob/user)
+	if (cooldown < world.time)
+		cooldown = world.time + 1800 //3 minutes
+		user.visible_message("<span class='warning'>[user] rotates a cogwheel on [src].</span>", "<span class='notice'>You rotate a cogwheel on [src], it plays a loud noise!</span>", "<span class='italics'>You hear cogwheels turning.</span>")
+		playsound(src, 'sound/magic/clockwork/ark_activation.ogg', 50, 0)
```

Comment:
Thoughts: this might be too annoying and spammy. This is a 30-second sound, whereas the most any other toy does is the nuke toy with around ~10 seconds. Playing the integration cog installation sound (`sound/machines/clockwork/integration_cog_install.ogg`, I think) and putting it on a smaller cooldown might be more fitting? Just my own idea though

Reply:
It's 11 seconds long

Reply:
Oh, I had this sound confused with the ark activation sequence. This is fair!